### ISSUE TITLE
[marathon] Marathon check is not disabling SSL validation

### ISSUE THREAD

**Issue by dlyub** [DataDog/dd-agent#2787]
Marathon yaml file looks as follows:

```
instances:
- url: https://<machine_fqdn>:8443
  disable_ssl_validation: true

init_config:
  default_timeout: 10
```

And the expectation was that the certificate validation would not be done, however agent info shows this:

```
marathon
--------
  - instance #0 [ERROR]: '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:590)'
  - Collected 0 metrics, 0 events & 1 service check
```

Tried using 'disable_ssl_validation: true' with haproxy yaml and it works as expected. Here's the full agent info output:

```
===================
Collector (v 5.8.5)
===================

  Status date: 2016-08-25 17:08:11 (15s ago)
  Pid: 12048
  Platform: Linux-3.10.0-327.10.1.el7.x86_64-x86_64-with-centos-7.2.1511-Core
  Python Version: 2.7.12, 64bit
  Logs: <stderr>, /var/log/datadog/collector.log

  Clocks
  ======

    NTP offset: Unknown (No response received from ns1.aspect.com.)
    System UTC time: 2016-08-25 17:08:31.552912

  Paths
  =====

    conf.d: /etc/dd-agent/conf.d
    checks.d: /opt/datadog-agent/agent/checks.d

  Checks
  ======

    network
    -------
      - instance #0 [OK]
      - Collected 15 metrics, 0 events & 1 service check

    zk
    --
      - instance #0 [OK]
      - Collected 25 metrics, 0 events & 2 service checks

    process
    -------
      - instance #0 [OK]
      - instance #1 [OK]
      - instance #2 [OK]
      - instance #3 [OK]
      - Collected 15 metrics, 0 events & 5 service checks

    ntp
    ---
      - Collected 0 metrics, 0 events & 1 service check

    disk
    ----
      - instance #0 [OK]
      - Collected 32 metrics, 0 events & 1 service check

    mesos_master
    ------------
      - instance #0 [OK]
      - Collected 73 metrics, 0 events & 2 service checks

    marathon
    --------
      - instance #0 [ERROR]: '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:590)'
      - Collected 0 metrics, 0 events & 1 service check


  Emitters
  ========

    - http_emitter [OK]

===================
Dogstatsd (v 5.8.5)
===================

  Status date: 2016-08-25 17:08:26 (5s ago)
  Pid: 12045
  Platform: Linux-3.10.0-327.10.1.el7.x86_64-x86_64-with-centos-7.2.1511-Core
  Python Version: 2.7.12, 64bit
  Logs: <stderr>, /var/log/datadog/dogstatsd.log

  Flush count: 1539
  Packet Count: 0
  Packets per second: 0.0
  Metric count: 1
  Event count: 0
  Service check count: 0

===================
Forwarder (v 5.8.5)
===================

  Status date: 2016-08-25 17:08:30 (3s ago)
  Pid: 12047
  Platform: Linux-3.10.0-327.10.1.el7.x86_64-x86_64-with-centos-7.2.1511-Core
  Python Version: 2.7.12, 64bit
  Logs: <stderr>, /var/log/datadog/forwarder.log

  Queue Size: 7670 bytes
  Queue Length: 1
  Flush Count: 5387
  Transactions received: 2332
  Transactions flushed: 2331
  Transactions rejected: 0

```

**Comment by masci** [comment]
Hi @dlyub and thanks for your report!
The marathon check doesn't support this feature, so a config param named `disable_ssl_validation` has no effect. 
Moving this to triage, will followup with a target release to ship this feature. In the meantime, if this is blocking, you can manually patch the check module on your installed agent, adding `verify=False` to the `request.get` call [here](https://github.com/DataDog/dd-agent/blob/master/checks.d/marathon.py#L61)

**Comment by gmmeyer** [comment]
This was fixed in #3140! It should be out in 5.12.
